package.json

```diff
@@ -7,6 +7,7 @@
   },
   "scripts": {
     "build": "node src/start build",
+    "postinstall": "node ./script/check-node-version.js",
```

Comment:
Does this make sense to run post or pre install? 🤔

Reply:
@Clark-W-Griswold decided on post install because it's not blocking and it'll leave the message front and center in the terminal when it's complete, also there's a dependency that's being used in the script that won't be there preinstall